homeassistant/components/onkyo/media_player.py

```diff
@@ -268,7 +270,13 @@ def update(self):
         volume_raw = self.command("volume query")
         mute_raw = self.command("audio-muting query")
         current_source_raw = self.command("input-selector query")
-        hdmi_out_raw = self.command("hdmi-output-selector query")
+        # If the following command is sent to a device with only one HDMI out,
+        # the display shows 'Not Available'.
+        # We avoid this by checking if HDMI out is supported
+        if self._hdmi_out_supported:
+            hdmi_out_raw = self.command("hdmi-output-selector query")
+        else:
+            hdmi_out_raw = False
```

Comment:
Please be consistent with typing as far as possible. If `hdmi_out_raw` is normally a list we should follow that here:
```suggestion
            hdmi_out_raw = []
```

Reply:
Of course. Change done.